### PR TITLE
[IMP] propose comment regarding eslint --fix command

### PR DESCRIPTION
**Pull request by legalsylvain**

### CODE REVIEW

File: website/Contribution/CONTRIBUTING.rst [hunk 1]
```diff
@@ -959,6 +959,15 @@ Javascript
 * Never add minified Javascript libraries
 * Use UpperCamelCase for class declarations
 
+You can autofix your javascript code. Once eslint installed, run the following lines locally: 
+
+```
+git clone https://github.com/OCA/pylint-odoo
+cd pylint-odoo
```
Comment (pedrobaeza) [inline website/Contribution/CONTRIBUTING.rst]: why this cd? better to put the path directly in the following command, isn't it?

Comment (legalsylvain) [reply website/Contribution/CONTRIBUTING.rst]: indeed. done.

**Comment by sbidoul** [comment]
Do we want to add more of these manual instructions, now that we are moving to pre-commit ?

**Comment by pedrobaeza** [comment]
I don't think so.

**Pull request closed without merging**